__tests__/api/costumes.test.js

```diff
@@ -16,28 +16,50 @@ const request = supertest(app)
 
 const pool = getPool();
 
-describe('GET api/costumes', () => {
+describe('/api/costumes', () => {
+    beforeEach(async () => {
+        await createTables(pool);
+    })
     afterAll(async () => {
         await pool.end()
     })
+    describe('GET api/costumes', () => {
+        it('should return all costumes when there is one in DB', async () => {
+            await createCostume(pool, getBallroomGown());
+            const response = await request.get('/api/costumes');
+            expect(response.status).toBe(200);
+            expect(response.body.costumes[0].name).toBe('ballroom gown');
+          })
     
-    it('should return all costumes when there is one in DB', async () => {
-        await createTables(pool);
-        await createCostume(pool, getBallroomGown());
-        const response = await request.get('/api/costumes');
-        expect(response.status).toBe(200);
-        expect(response.body.costumes[0].name).toBe('ballroom gown');
-      })
+          it('should return all costumes when there is multiple in DB', async () => {
+            await createCostume(pool, getBallroomGown());
+            await createCostume(pool, getButtlessChaps());
+            await createCostume(pool, getBonnet());
+            const response = await request.get('/api/costumes');
+            expect(response.status).toBe(200);
+            expect(response.body.costumes[0].name).toBe('ballroom gown');
+            expect(response.body.costumes[1].name).toBe('buttless chaps');
+            expect(response.body.costumes[2].name).toBe('bonnet');
+          })
+    })
+    describe('POST /api/costumes', () => {
+        it('should succeed in POST request', async () => {
+            const response = await request.post('/api/costumes').send(getBonnet());
+            expect(response.status).toBe(200);
+            expect(response.body.costume.name).toBe('bonnet');
```

Comment:
Prefer not testing hard-coded values if possible
```suggestion
            const bonnet = getBonnet();
            const response = await request.post('/api/costumes').send(bonnet);
            expect(response.status).toBe(200);
            expect(response.body.costume.id).toBe(bonnet.id);
            expect(response.body.costume.name).toBe(bonnet.name);
```

Reply:
Yes! I see your point.

Reply:
The only thing is that I don't think line 50 will work on your edit because bonnet will not have an id until it is added to the db (getBonnet just creates the object without an id yet). I could check another property on bonnet in addition to name. Or I can have a hard-coded expect for the id number the bonnet should have in the response body based on the order the costume was added, but I don't know if that might be flaky or not.